app/templates/components/addon-size.hbs

```diff
@@ -0,0 +1,68 @@
+<div class="addon-size {{@class}}">
```

Comment:
This should be `<div class="addon-size" ...attributes>` which will put html attributes from the calling side here (details: https://guides.emberjs.com/release/components/component-arguments-and-html-attributes/#toc_html-attributes)